src/patternfly/components/ClipboardCopy/examples/clipboard-copy-no-expansion-example.hbs

```diff
@@ -1,18 +1,18 @@
 Read-Only
 <br>
 {{#> clipboard-copy}}
-  {{#> clipboard-copy-group}}
-    {{#> clipboard-copy-group-text}}This text is read-only{{/clipboard-copy-group-text}}
-    {{#> clipboard-copy-group-copy}}{{/clipboard-copy-group-copy}}
-  {{/clipboard-copy-group}}
+	{{#> clipboard-copy-group}}
+		{{#> clipboard-copy-group-text}}This text is read-only{{/clipboard-copy-group-text}}
+		{{#> clipboard-copy-group-copy}}{{/clipboard-copy-group-copy}}
+	{{/clipboard-copy-group}}
 {{/clipboard-copy}}
 <br>
 <br>
 Editable
 <br>
 {{#> clipboard-copy}}
 	{{#> clipboard-copy-group}}
-	{{#> clipboard-copy-group-text clipboard-copy-group-text--type="textarea" clipboard-copy-group-text--modifier="pf-m-editable"}}This text is editable{{/clipboard-copy-group-text}}
+	{{#> clipboard-copy-group-text clipboard-copy-group-text--modifier="pf-m-editable" clipboard-copy-group-text--type="textarea"}}This text is editable{{/clipboard-copy-group-text}}
```

Comment:
Could this be a textarea or input[type="text"]? Or will it always be a textarea if it's editable? The textarea is behaving differently - has a different minimum width and spaces in the textarea show up before the text. I wonder if you could just use `contenteditable` instead?

Reply:
contenteditable works!